integration-test/src/test/java/io/lumify/it/VertexIntegrationTest.java

```diff
@@ -49,7 +49,7 @@ public void testSearchVisibleWithQueryString() throws ApiException {
         List<ClientApiVertex> vertices;
 
         // matches all visible
-        vertices = helper.vertexApi.vertexSearch(PROPERTY_QUERY_STRING, EMPTY_FILTER, null, null, null, null,
+        vertices = helper.vertexApi.vertexSearch(PROPERTY_QUERY_STRING + "*", EMPTY_FILTER, null, null, null, null,
```

Comment:
Sounds good. I just wanted to make sure that it was the test itself that was broken and not the code in question.